js/okx.js

```diff
@@ -654,6 +654,24 @@ module.exports = class okx extends Exchange {
         });
     }
 
+    typeToExchangeType (type) {
+        type = type.toUpperCase ();
+        if (type === 'FUTURE') {
```

Comment:
I would refrain from doing this and from adding this method, let's just use a mapping from the options instead with safeStringUpper.